src/app/redux/UserSaga.js

```diff
@@ -67,6 +68,19 @@ export function* watchRemoveHighSecurityKeys() {
     yield* takeLatest('@@router/LOCATION_CHANGE', removeHighSecurityKeys);
 }
 
+function* getLatestFeedPrice() {
+    try {
+        const history = yield call([api, api.getFeedHistoryAsync]);
+        if (history) {
```

Comment:
We don't need this if statement anymore, because if the `yield call` fails, the catch will get it...

Reply:
Is it possible `yield call` can return blank without throwing an exception?

Reply:
Cannot tell - where is this endpoint documented?

Reply:
I'm not sure it's documented anywhere; https://github.com/steemit/steem/blob/bb8ca476f33c08434274f2d76333fcb74cfa58b8/libraries/plugins/apis/database_api/database_api.cpp#L158 is where the API is defined and https://github.com/steemit/steem/blob/62b48877d9f731c3fe00ef818e3324a0a3de3e63/libraries/chain/database.cpp#L3082 is where the structure is maintained